src/index.js

```diff
@@ -252,6 +260,13 @@ export default async function fetch(url, options_) {
 					response = new Response(body, responseOptions);
 					resolve(response);
 				});
+				raw.once('end', () => {
+					// some old IIS servers return zero-length OK deflate responses, so 'data' is never emitted.
+					if (!response) {
```

Comment:
- I would like to see a comment with link to explain special handling of deflate response, so we have a source for the code, see my comment above that explains the line `if ((chunk[0] & 0x0F) === 0x08)`